test/unit/gateways/orbital_test.rb

```diff
@@ -269,6 +291,9 @@ def test_network_tokenization_credit_card_data
 
   def test_schema_for_soft_descriptors_with_network_tokenization_credit_card_data
```

Comment:
@jknipp I updated this unit test to bring in the fields with L2/L3 data. Refactored the assertion to ensure it is valid to XSD instead of just matching specific fields.

Reply:
This test could probably be renamed since it's no longer specific to network tokenization and soft descriptors